xml/remote_administration.xml

```diff
@@ -298,42 +296,45 @@ nodeattr [-f genders] -k
     Check permissions, owner and file type of the key file located under
     <filename>/etc/munge/munge.key</filename> on the local system:
    </para>
-   <screen># stat --format "%F %a %G %U %n" /etc/munge/munge.key</screen>
+<screen># stat --format "%F %a %G %U %n" /etc/munge/munge.key</screen>
    <para>The settings should be:</para>
-   <screen>400 regular file munge munge /etc/munge/munge.key</screen>
+<screen>400 regular file munge munge /etc/munge/munge.key</screen>
    <para>Get md5sum of munge.key:</para>
-   <screen># md5sum /etc/munge/munge.key</screen>
+<screen># md5sum /etc/munge/munge.key</screen>
    <para>Copy key to list of nodes:</para>
-   <screen># pdcp -R ssh -w &lt;hostlist&gt; /etc/munge/munge.key
+<screen># pdcp -R ssh -w &lt;hostlist&gt; /etc/munge/munge.key
    /etc/munge/munge.key </screen>
    <para>Check key settings in remote host:</para>
-   <screen>pdsh -R ssh -w &lt;hostlist&gt; stat --format \"%F %a %G %U %n\"
-   /etc/munge/munge.key
-   pdsh -R ssh -w &lt;hostlist&gt; md5sum /etc/munge/munge.key
-   </screen>
-   <para>make sure they match.</para>
+<screen>pdsh -R ssh -w &lt;hostlist&gt; stat --format \"%F %a %G %U %n\"
+/etc/munge/munge.key
+pdsh -R ssh -w &lt;hostlist&gt; md5sum /etc/munge/munge.key</screen>
+   <para>Make sure that they match.</para>
   </sect2>
   <sect2>
-   <title>Enabling and starting MUNGE</title>
+   <title>Enabling and starting Munge</title>
    <para>
-    <emphasis>munged</emphasis> needs to be run on all nodes where munge
-    authentication is to take place. If munge is used for authentication
-    across the network it needs to run on each side of the communication.
+    <systemitem class="daemon">munged</systemitem> needs to be run on all nodes
+    where Munge authentication will take place. If Munge is used for
+    authentication across the network, it needs to run on each side of the
+    communications link.
    </para>
    <para>
-    To start the service and make sure it is started after every reboot, run</para>
-    <screen>systemctl enable munge.service
-    systemctl start munge.service</screen>
-    <para>on each node. To perform the same on multiple nodes, pdsh may be employed
-    as well.
+    To start the service and make sure it is started after every reboot, on each
+    node, run:
+   </para>
+<screen>systemctl enable munge.service
+systemctl start munge.service</screen>
+    <para>
+     To perform the same on multiple nodes, you can also use
+     <command>pdsh</command>.
    </para>
   </sect2>
  </sect1>
  <sect1 xml:id="sec-remote-mrsh">
-  <title>mrsh/mrlogin &mdash; remote login using MUNGE authentication</title>
+  <title>mrsh/mrlogin &mdash; remote login using Munge authentication</title>
```

Comment:
MUNGE is the correct spelling (see the homepage of the project-- https://dun.github.io/munge/ ):
```suggestion
  <title>mrsh/mrlogin &mdash; remote login using MUNGE authentication</title>
```

Please rather fix other occurrences of this.